blog/_posts/2023-05-30-new-class-tscatter.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "New class TScatter"
+layout: archive
+author: Olivier Couet
+---
+
+<center>
+   <img src="{{'/assets/images/scatter.png' | relative_url}}" alt="TScatter" style="width: 60%" width="1600" height="749" />
+</center>
+
+2D Scatter plots are a very popular way to represent scientific data. [Many scientific plotting
+packages](https://matplotlib.org/stable/gallery/shapes_and_collections/scatter.html#sphx-glr-gallery-shapes-and-collections-scatter-py)
+have this functionality. ROOT itself offers this kind of tools via:
+
+  - [The option `SCAT` to draw 2D histograms](https://root.cern/doc/master/classTHistPainter.html#HP11).
```

Comment:
BTW, someone raised the point that in English, Scat might be seen as an alternate term for [feces, particularly of wild carnivores](https://en.wikipedia.org/wiki/Feces#Feces_of_animals). See https://en.wikipedia.org/wiki/Scat
So maybe another option would be better?

Reply:
:-) Well it is like that now we cannot change it. My idea, ideally, is to make this option "obsolete" anyway as it is completely counterintuitive.